package-examples/ghost/fn/validate-blog-host.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: fn.kpt.dev/v1alpha1
+kind: StarlarkRun
+metadata:
+  name: validate-blog-host
+source: |-
+  def validate_host(resources):
+    errMsg = "Blog's hostname must be specified in fn/update-blog-host.yaml"
+    placeHolderHostname = "example.com"
+    for resource in resources:
+      # this is an abstract package, so ignore host name validation
```

Comment:
We probably should have a first-class way to query this in our sdks.

Reply:
Yes, validation functions needs this bit, so a helper in SDK will help.

/cc @yuwenma